napi-inl.h

```diff
@@ -4303,6 +4342,363 @@ inline void AsyncWorker::OnWorkComplete(Napi::Env /*env*/, napi_status status) {
 }
 
 #if (NAPI_VERSION > 3 && !defined(__wasm32__))
+////////////////////////////////////////////////////////////////////////////////
+// ThreadSafeFunctionEx<ContextType,DataType,CallJs> class
+////////////////////////////////////////////////////////////////////////////////
+
+// Starting with NAPI 4, the JavaScript function `func` parameter of
```

Comment:
Should this say starting with NAPI 5 , as that is what the #if does.

Reply:
Good catch; this has been addressed [here](https://github.com/nodejs/node-addon-api/blob/3405b2bf39793a573d420584ea81a411e5127634/napi-inl.h#L4401-L4402).